linuxbrew-core/Dockerfile

```diff
@@ -1,14 +1,18 @@
 FROM ubuntu
 MAINTAINER Shaun Jackman <sjackman@gmail.com>
 
+ENV DEBIAN_FRONTEND noninteractive
+
 RUN apt-get update
-RUN apt-get install -y curl g++ gawk m4 make patch ruby tcl
+RUN apt-get install -qq curl g++ gawk m4 make patch ruby tcl git
 
 RUN useradd -m linuxbrew
 RUN echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
 
 USER linuxbrew
-WORKDIR /home/linuxbrew
-ENV PATH /home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH
-RUN yes |ruby -e "$(curl -fsSL https://raw.github.com/Homebrew/linuxbrew/go/install)"
```

Comment:
I want to test the `go/install`, which is the recommended installation method by http://brew.sh/linuxbrew